sdk/samples/iot/CMakeLists.txt

```diff
@@ -22,13 +22,8 @@ if(NOT OpenSSL_FOUND)
 endif()
 
 # Azure IoT Samples Library
-add_library (${TARGET_NAME}
-  ${CMAKE_CURRENT_LIST_DIR}/src/sample_sas_utility.c
-)
-
-target_include_directories (${TARGET_NAME}
-  PUBLIC
-  ${CMAKE_CURRENT_LIST_DIR}/inc
+add_library (az_iot_samples_common
```

Comment:
This change overwrote parts of the changes from https://github.com/Azure/azure-sdk-for-c/pull/961

Specifically, using `{TARGET_NAME}`, from `set(TARGET_NAME "az_iot_samples_common")`

Reply:
Please take a look at this file (from the above linked PR) and retain pieces that make sense still with your change: https://github.com/ahsonkhan/azure-sdk-for-c/blob/7ad50097b2eb8708345970440306f4b59b3551b6/sdk/samples/iot/CMakeLists.txt

Reply:
looks like my commit from yesterday did this.  fixing.